__snapshots__/helpers_spec.js

```diff
@@ -7,4 +7,3 @@ exports['lib/helpers addPageAnchors is noop if no headings found 1'] = '<p>foo</
 exports['lib/helpers addPageAnchors does not wrap with <html> 1'] = '<h1 id="bar" class="article-heading">foo<a class="article-anchor" href="#bar" aria-hidden="true"></a></h1>'
 
 exports['lib/helpers addPageAnchors does not wrap fragment in html tag 1'] = '<p>foo</p>'
```

Comment:
is your editor set to not add a blank line at the end of the file?

Reply:
@CypressCecelia If you don't already have eslint in your text editor, it will suggest these types of lint / style conventions inline in your editor before committing.

You may have `trimFinalNewLines` on in your editor? https://github.com/Microsoft/vscode/issues/12076#issuecomment-340037747

![Screen Shot 2020-03-13 at 4 12 04 PM](https://user-images.githubusercontent.com/1271364/76609222-67fcf000-6545-11ea-9e6d-c799960d91a1.png)

Reply:
Weird, it's not enabled in my settings. I'll add the eslint extension to my editor, thanks.